CMakeLists.txt

```diff
@@ -232,9 +232,14 @@ if(USE_TENSORRT)
 endif()
 if(WITH_TENSORFLOW2_LIB)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DDLR_TENSORFLOW2")
-    include_directories("${WITH_TENSORFLOW2_LIB}/include")
+    if(NOT WITH_TENSORFLOW2_LIB)
```

Comment:
This IF will always be False because you are inside `if(WITH_TENSORFLOW2_LIB)` block.

Reply:
Fixed.